catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v1_1_0/catalog/source/WfsSourceTest.java

```diff
@@ -0,0 +1,1138 @@
+      "<?xml version=\"1.0\"?>"
+          + "<xs:schema xmlns:xs=\"http://www.w3.org/2001/XMLSchema\" "
+          + "xmlns:gml=\"http://www.opengis.net/gml\" "
+          + "targetNamespace=\"http://www.opengis.net/gml\">"
+          + "<xs:element name=\"shiporder\">"
+          + "<xs:complexType>"
+          + "<xs:sequence>"
+          + "<xs:element name=\"orderperson\" type=\"gml:FakeGmlProperty\"/>"
+          + "<xs:element name=\"orderdog\" type=\"gml:FakeGmlProperty\"/>"
+          + "</xs:sequence>"
+          + "</xs:complexType>"
+          + "</xs:element>"
+          + "<xs:complexType name=\"FakeGmlProperty\" />"
+          + "</xs:schema>";
+
+  private static final String GML_IMPORT_SCHEMA =
+      "<?xml version=\"1.0\"?>"
+          + "<xs:schema xmlns:xs=\"http://www.w3.org/2001/XMLSchema\" "
+          + "xmlns:gml=\"http://www.opengis.net/gml\" "
+          + "targetNamespace=\"http://www.opengis.net/gml\">"
+          + "<xs:import namespace=\"http://www.opengis.net/gml\" schemaLocation=\"http://schemas.opengis.net/gml/2.1.2/feature.xsd\"/>"
+          + "<xs:element name=\"shiporder\">"
+          + "<xs:complexType>"
+          + "<xs:sequence>"
+          + "<xs:element name=\"orderperson\" type=\"gml:FakeGmlProperty\"/>"
+          + "</xs:sequence>"
+          + "</xs:complexType>"
+          + "</xs:element>"
+          + "<xs:complexType name=\"FakeGmlProperty\" />"
+          + "</xs:schema>";
+
+  private static final String POLYGON_WKT = "POLYGON ((30 10, 10 20, 20 40, 40 40, 30 10))";
+
+  private static final String ORDER_PERSON = "orderperson";
+
+  private static final String ORDER_DOG = "orderdog";
+
+  private static final Integer MAX_FEATURES = 10;
+
+  private static final Integer ONE_FEATURE = 1;
+
+  private static final Integer TWO_FEATURES = 2;
+
+  private static final String SRS_NAME = "EPSG:4326";
+
+  private static final String LITERAL = "literal";
+
+  private static final String EXT_PREFIX = "ext.";
+
+  private static final Comparator<QueryType> QUERY_TYPE_COMPARATOR =
+      (queryType1, queryType2) -> {
+        String typeName1 = queryType1.getTypeName().get(0).getLocalPart();
+        String typeName2 = queryType2.getTypeName().get(0).getLocalPart();
+        return typeName1.compareTo(typeName2);
+      };
+
+  private final GeotoolsFilterBuilder builder = new GeotoolsFilterBuilder();
+
+  private Wfs mockWfs = mock(Wfs.class);
+
+  private WFSCapabilitiesType mockCapabilites = new WFSCapabilitiesType();
+
+  private WfsFeatureCollection mockFeatureCollection = mock(WfsFeatureCollection.class);
+
+  private BundleContext mockContext = mock(BundleContext.class);
+
+  private List<QName> sampleFeatures;
+
+  private WfsUriResolver wfsUriResolver = new WfsUriResolver();
+
+  private WfsSource source;
+
+  private AvailabilityTask mockAvailabilityTask = mock(AvailabilityTask.class);
+
+  private EncryptionService encryptionService = mock(EncryptionService.class);
+
+  public void setUp(
```

Comment:
shouldn't this be annotated with `@Before` ?  It seems odd that we just call it at the beginning of every test when that's the purpose of `@Before`

Reply:
It's called with different parameters for each unit test.